e2e/e2e-scenario.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+source e2e-helper.sh
+
+debug() {
+    local retval
+    retval=0
+    mkdir -p $SCENARIO_NAME-logs
+    INSTANCE_ID="$(az vmss list-instances --name $VMSS_NAME -g $MC_RESOURCE_GROUP_NAME | jq -r '.[0].instanceId')"
+    PRIVATE_IP="$(az vmss nic list-vm-nics --vmss-name $VMSS_NAME -g $MC_RESOURCE_GROUP_NAME --instance-id $INSTANCE_ID | jq -r .[0].ipConfigurations[0].privateIpAddress)"
+    set +x
+    SSH_KEY=$(cat ~/.ssh/id_rsa)
```

Comment:
Is there a reason this can't be done with `run-command`?

Reply:
it's like 10-20 minutes slower. we used run command command before this